.ci_support/build_all.py

```diff
@@ -47,6 +87,20 @@ def build_all(recipes_dir, arch):
     config = conda_build.api.Config(
         variant_config_files=variant_config_files, arch=arch,
         exclusive_config_file=exclusive_config_file, channel_urls=channel_urls)
+    return config
+
+def build_folders(recipes_dir, folders, arch, channel_urls):
+
+    index_path = os.path.join(sys.exec_prefix, 'conda-bld')
+    try:
+        os.mkdir(index_path)
+    except FileExistsError:
+        pass
```

Comment:
I think that `os.makedirs(index_path, exist_ok=True)` is cleaner than the try-except here